### DISCUSSION TITLE
Trouble Integrating Moodle with GitHub Classroom LTI

### DISCUSSION THREAD

**Discussion by jrieffel** [community/community#51406] [GitHub Education]
My institution has followed all the steps to integrate our Moodle instance with GitHub classroom, and it *kind of* works, but we are struggling to get Moodle rosters to be imported into GitHub classroom.  Has anyone had success in this matter?

**Comment by samus-aran** [comment]
Hi @jrieffel 👋 

What errors are you experiencing? Screenshot would be helpful but do exclude any personal/institution identifying information.

**Comment by jrieffel** [comment]
Instead of importing the roster from moodle it imports a single record
named "710".  There are 43 students on the roster.

[image: image.png]
